system/cronvar.py

```diff
@@ -114,14 +114,12 @@ class CronVar(object):
     """
         CronVar object to write variables to crontabs.
 
-        user      - the user of the crontab (defaults to root)
+        user      - the user of the crontab (defaults to None)
```

Comment:
In which case it defaults to the ansible_user.  It doesn't default to a literal "None" nor does it default to no user. Change this to "defaults to ansible_user" and I'll be fine with the PR.